package.json

```diff
@@ -55,22 +58,22 @@
     "@typescript-eslint/eslint-plugin": "^5.58.0",
     "@typescript-eslint/parser": "^5.58.0",
     "beachball": "^2.33.2",
-    "eslint": "~8.15.0",
     "eslint-config-prettier": "8.1.0",
     "eslint-plugin-import": "^2.27.5",
-    "jest": "^29.4.1",
+    "eslint": "~8.15.0",
```

Comment:
I assume this re-ordering happened by `yarn add` ?

Reply:
Yes. I reverted as this is unnecessary.